src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,8 @@
+prison.api.endpoint.url: https://api-dev.prison.service.justice.gov.uk
+
+spring:
```

Comment:
Rather than using the embedded public key, which only works against locally running instances, shouldn't we be pointing it at **https://sign-in-dev.hmpps.service.justice.gov.uk/auth/.well-known/jwks.json**?